beetsplug/discogs.py

```diff
@@ -557,7 +557,8 @@ def get_track_info(self, track, index, divisions):
         title = track['title']
         if self.config['index_tracks']:
             prefix = ', '.join(divisions)
-            title = ': '.join([prefix, title])
+            if prefix != '':
+                title = ': '.join([prefix, title])
```

Comment:
Not your change, obviously, but it occurs to me that a clearer way to write this would be `'{}: {}'.format(prefix, title)`.